install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -14,6 +14,14 @@ ALTER TABLE `PREFIX_product_attribute` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 ALTER TABLE `PREFIX_product` ADD `mpn` VARCHAR(40) NULL AFTER `upc`;
 
 /* Delete price display precision configuration */
+
+DELETE `PREFIX_configuration` WHERE name = 'PS_PRICE_DISPLAY_PRECISION';
+
+# improve performance of lookup by product reference/product_supplier avoiding full table scan
```

Comment:
This is not an SQL comment

Reply:
@PierreRambaud are you sure? it's been a mysql single line comment since v5.5 https://dev.mysql.com/doc/refman/5.5/en/comments.html
as is double dash --
maybe you're talking about ANSI SQL?

Reply:
either way changed it to be consistent with other comment styles

Reply:
unfortunately we are supposed to support 5.1 mysql version :(

Reply:
wow that's old, mysql official documentation doesn't even list anything past 5.5 :(

but hey a copy of mysql docs says # has been there since 5.0 http://download.nust.na/pub6/mysql/doc/refman/5.0/en/comments.html

Reply:
Ohh, my bad, always use # in bash only :sweat_smile:

Reply:
well i made it consistent anyways so /**/ all the way 😄